src/scancode/cli.py

```diff
@@ -255,26 +262,42 @@ def scancode(ctx, input, output_file, copyright, license, package,
     Error and progress is printed to stderr.
     """
 
+    possible_scans = OrderedDict([
```

Comment:
I think I get where you are going with this, but we end having eventually three bits of data: the scan function, the name of the scan and its option flag as true or false.
This likely reaches the point where a small object would be best and more explicit and would avoid any indexed property access or zipping for parallel sequences realignments.